SUPPORT.md

```diff
@@ -0,0 +1,37 @@
+# Player and Server Admin Support
+
+Are you a player or server admin having an issue installing or using Fabric Loader, Fabric API or Fabric mods in general?
+
+- [Fabric Wiki - Player Tutorials](https://fabricmc.net/wiki/player:tutorials:start)
+- [Official Fabric Discord - `#player-support`](https://discord.gg/v6v4pMv)
+
+You can also [create a new discussion](https://github.com/orgs/FabricMC/discussions/new?category=player-support) on the GitHub Organization page.
+
+# Modpack Developer Support
+
+Are you a modpack developer having problems trying to create a modpack using Fabric?
+
+<!-- Can be uncommented when the pages are created  -->
+<!-- - [Fabric Wiki - Modpack Guides](https://fabricmc.net/wiki/modpack:start)
+- [Fabric Wiki - Creating A Modpack Using Packwiz](https://fabricmc.net/wiki/modpack:creating_packwiz)
+- [Fabric Wiki - Creating A Modpack Using ATLauncher](https://fabricmc.net/wiki/modpack:creating_atlauncher) -->
+- [Official Fabric Discord - `#player-support` and `#mod-dev` channels](https://discord.gg/v6v4pMv)
```

Comment:
This whole section is a bit pointless atm, can add it back if/when these wiki pages are created.